.github/workflows/maven.yml

```diff
@@ -15,12 +15,25 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
-    - name: Set up JDK 11
-      uses: actions/setup-java@v3
-      with:
-        java-version: '11'
-        distribution: 'temurin'
-        cache: maven
-    - name: Build with Maven
-      run: mvn -B package --file pom.xml
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+          cache: maven
+      - name: Build with Maven
+        run: mvn -B package --file pom.xml
+
+  checkstyle:
+    name: checkstyle linting
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dbelyaev/action-checkstyle@v0.6.1
+        with:
+          github_token: ${{ secrets.github_token }}
```

Comment:
Is github_token mandatory over here?

Reply:
this only to catch info for PR but it's not mandatory even when I deleted this line things still work.

Reply:
I dont think that is required . Remove it if it is not necessary.